codex/proof/slotblocks.nim

```diff
@@ -0,0 +1,110 @@
+import std/sequtils
```

Comment:
This isn't needed, just use `slotsBuilder.buildSlot(idx)` and everything will be taken care of.